src/entities/trades/swapr-v3/pools.ts

```diff
@@ -0,0 +1,168 @@
+import { Currency, Token } from '@uniswap/sdk-core'
+import { POOL_DEPLOYER_ADDRESS, baseTokens } from './constants'
+import { computePoolAddress } from './utils/computePoolAddress'
+import { Pool } from './entities/pool'
+import { getPoolsContract } from './contracts'
+import { ChainId } from '../../../constants'
+
+const getBaseTokens: Token[] = baseTokens.map(
+  ({ address, decimals, symbol, name }) => new Token(ChainId.GNOSIS, address, decimals, symbol, name)
+)
+
+const WXDAI_ADDRESS = '0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d'
```

Comment:
There is already a constant `WXDAI` token entity on the SDK. It's on `token.ts` file. Let's is it instead of this,

Reply:
Yes, my bad. Tks